<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html><head><!-- saved from url=(0030)https://dongxiaozhu.github.io/ -->

















<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><title>Dongxiao Zhu</title></head><body>
    <div style="text-align: center; font-family: Arial;">
      <div style="text-align: center;"></div>
      <div style="text-align: center;">
        <div style="text-align: center;">&nbsp; <span style="font-size: 12pt;"><a href="mailto:dzhu@wayne.edu"><span style="font-weight: bold;">Dongxiao Zhu</span></a></span><br>
          <span style="font-size: 12pt;"></span><a href="http://www.cs.wayne.edu/" target="_blank">Department of
              Computer Science</a> <br>
5057 Woodward Ave. Suite 14101.3<br style="font-size: 12pt;">
          <span style="font-size: 12pt;"><a href="http://www.wayne.edu/">Wayne State University</a>, Detroit, MI 48202</span><br>
        </div>
        <a href="mailto:mailto:dzhu@wayne.edu"><span style="font-size: 12pt;"></span></a></div>
      <hr style="width: 100%; height: 2px;"></div>
    <span style="font-size: 12pt; font-family: Arial;"><span style="font-weight: bold;">Short Bio:</span> <br>
<br>
Dongxiao Zhu is currently an Associate Professor at Department of
Computer Science, Wayne State University. He received the B.S. from
Shandong University (1996), the M.S. from Peking University (1999) and
the Ph.D. from University of Michigan (2006). Dongxiao Zhu's recent
research interests are in <span style="font-weight: bold;">Machine Learning and Applications</span>
in health informatics, natural language processing, medical imaging,
recommender systems, cybersecurity, and other data rich domains with
emphasis on S &amp; P, adversarial robustness, explainability and
fairness. Dr. Zhu is the
Director of <a style="font-weight: bold;" href="ai.wayne.edu" target="_blank">Wayne AI Research Group</a> and Director of <a style="font-weight: bold;" href="https://sites.google.com/view/mlpa/mainpage" target="_blank">Trustworthy AI</a><a style="font-weight: bold;" href="https://sites.google.com/view/mlpa/mainpage" target="_blank"><span style="font-weight: bold;"> Lab</span></a> at Wayne State University. </span><span style="color: rgb(0, 0, 0); font-size: 12pt; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; font-family: Arial;">He
has published over 80 peer-reviewed publications and served on program
committees (Senior PC or PC) of flagship AI conferences (</span><span style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: 12pt; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px;"><span style="font-family: Arial;"></span><span style="font-style: italic; font-family: Arial;">NuerIPS, ICML, AAAI, IJCAI, ACL, EMNLP</span></span><span style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: 12pt; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px;"><span style="font-style: italic; font-family: Arial;">, AMIA, MICCAI</span></span><span style="color: rgb(0, 0, 0); font-size: 12pt; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; font-family: Arial;">) and<span>&nbsp;</span></span><span style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: 12pt; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px;"><span style="font-family: Arial;">of premier biomedical informatics journals (<span style="font-style: italic;">Bioinformatics</span></span><span style="font-style: italic; font-family: Arial;">, Nucleic Acids Research, TCBB, Scientific Reports, BMC Genomics, Plos One<span>&nbsp;</span></span><span style="font-family: Arial;">and</span><span style="font-style: italic; font-family: Arial;"><span>&nbsp;</span>Frontiers in Genetics</span><span style="font-family: Arial;">).
Dr. Zhu's research has been supported by NIH, NSF and private agencies.
Dr. Zhu has advised numerous students at undergraduate, graduate and
postdoctoral levels and his teaching interest lies in programming
language, data structures and algorithms, machine learning and data
science.</span></span><br>
<br>
<span style="font-size: 12pt; font-family: Arial;"></span><span style="font-family: Arial;"><span style="font-weight: bold;"></span></span><span style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: 12pt; font-style: normal; font-weight: bold; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px;"><span style="font-family: Arial;"><br>
</span></span><span style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: 12pt; font-style: normal; font-weight: bold; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px;"><span style="font-family: Arial;"></span></span><span style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: 12pt; font-style: normal; font-weight: bold; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px;"><span style="font-family: Arial;">Pre-prints</span></span><br>
<ul style="font-family: Arial;">
  <li>Qiang, Y., Kumar, S. T. S., Brocanelli, M., &amp; Zhu, D. (2021). <a href="https://arxiv.org/pdf/2101.05624.pdf" target="_blank">Adversarially robust and explainable model compression with on-device personalization for NLP applications</a>. arXiv preprint arXiv:2101.05624.<br>
  </li>
  <li>Li, X, Pan, D,Li, X and Zhu, D (2020). <a href="https://dongxiaozhu.github.io/RegularizeMiniBatches.pdf" target="_blank">Regularize SGD training via aligning min-batches</a>.arXiv:2002.09917 [cs.LG]. <br>
<span style="font-size: 12pt; color: black;"></span></li>

</ul>
<h4 style="font-family: Arial;">Selected Recent Publications (<a href="https://dongxiaozhu.github.io/pubs.html" target="_blank">entire period</a>):&nbsp;</h4>
<span style="color: rgb(0, 0, 0); font-family: &quot;Times New Roman&quot;; font-size: 12pt; font-style: normal; font-weight: bold; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px;"><span style="font-family: Arial;"></span></span><span style="font-family: Arial;">2021</span><br style="font-family: Arial;">


<ul style="font-family: Arial;">
  <li>Wang, L., &amp; Zhu, D. (2021). <a href="https://arxiv.org/abs/1907.12508" target="_blank">Tackling multiple ordinal regression problems: sparse and deep multi-task learning approaches</a>. arXiv preprint arXiv:1907.12508, accepted by <span style="font-style: italic;">Data Mining and Knowledge Discovery (DMKD).</span><br>
</li>
  <li>Li, X., Pan, D. and Zhu, D., (2021) <a href="https://arxiv.org/pdf/2006.13555.pdf">Defending against adversarial attacks on medical imaging AI system, classification or detection?</a> To appear in the proceedings of <span style="font-style: italic;">IEEE International Symposium on Biomedical Imaging (ISBI-21)</span>, virtual conference.</li>

  <li>Li, X, Li, X, Pan,D and
Zhu, D (2021) <a href="https://arxiv.org/pdf/2012.07688.pdf" target="_blank">Improving adversarial robustness via probabilistically
compact loss with logit constraints</a>. To appear in the proceedings of <span style="font-style: italic;">Thirty-Five
AAAI Conference on Artificial Intelligence (AAAI-21), </span>virtual conference. <a href="https://github.com/xinli0928/PC-LC">Code</a>
  </li>
</ul>
<span style="font-family: Arial;">2020</span><br style="font-family: Arial;">
<ul style="font-family: Arial;"><li>Li, X and Zhu, D (2020) <a href="https://arxiv.org/abs/2004.03042">COVID-MobileXpert: On-Device COVID-19 Screening using Snapshots of Chest X-Ray</a>. To appear in the proceedings of <span style="font-style: italic;">2020 International Conference on Bioinformatics and Biomedicine (BIBM-20)</span>. <a href="https://github.com/xinli0928/COVID-Xray" target="_blank">Code</a> <a href="https://arxiv.org/abs/2004.03042">BibTeX</a></li>

  <li>Pan, D, Li, X, Li, X and Zhu, D (2020) <a href="https://www.ijcai.org/Proceedings/2020/0373.pdf" target="_blank">Explainable
recommendation via interpretable feature mapping and evaluating
explainability</a>. In the proceedings of <span style="font-style: italic;">29th International Joint Conference on Artificial Intelligence (IJCAI-20)</span>, Yokohama, Japan. <a href="https://github.com/pd90506/AMCF">Code</a> <a href="https://dblp.org/rec/conf/ijcai/PanLLZ20.html?view=bibtex">DBLP</a>&nbsp; </li>
<li>Qiang, Y, Li, X and Zhu, D (2020) <a href="https://dongxiaozhu.github.io/IJCNN20-ABSA.pdf" target="_blank">Toward tag-free aspect based sentiment analysis: a multiple attention network approach</a>. in the proceedings of <span style="font-style: italic;">International Joint Conference on Neural Networks (IJCNN-20)</span>, Glasgow, Scotland, UK. <a href="https://github.com/qiangyao1988/Toward-tag-free-ABSA">Code</a> <a target="_blank" href="https://scholar.googleusercontent.com/scholar.bib?q=info:va7jZJLJNgYJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wELzGkK3sStg:AAGBfm0AAAAAX1TpUtgzmFpn2J7dr-L0tLAlepRBZ8hK&amp;scisig=AAGBfm0AAAAAX1TpUoEgmuoYLpoXZrGUBKh6s4qMKbei&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en&amp;scfhb=1">BibTeX</a>
</li>
  
  <li>Li, X., Cao, R., &amp; Zhu, D. (2020). <a href="https://arxiv.org/abs/1906.05190" target="_blank">Vispi: Automatic visual perception and interpretation of chest X-rays</a>. In the proceedings of the <span style="font-style: italic;">Medical Imaging with Deep Learning (MIDL-20) </span>conference, Montreal, CA. <a href="https://github.com/xinli0928/COVID-Xray">Code</a> <a href="https://openreview.net/forum?id=otswIbmgYA">BibTeX</a>
  </li>
  <li>Li, X, Li, X, Pan,D and Zhu, D (2020) <a href="https://aaai.org/ojs/index.php/AAAI/article/view/5907">On the learning behavior of
logistic and softmax losses for deep neural networks</a>. In the proceedings of <span style="font-style: italic;">Thirty-Fourth
AAAI Conference on Artificial Intelligence (AAAI-20)</span>, New York, USA. <a href="https://github.com/Dichoto/LGL-INR">Code</a> <a href="https://dblp.org/rec/conf/aaai/LiLPZ20.html?view=bibtex">DBLP</a><br>
<span style="font-size: 12pt; color: black;"></span></li>
  <li>Li, X, Zhu, D* and Levy, P (2020) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233047/" target="_blank">Predicting clinical outcomes
with patient stratification via deep mixture neural networks</a>. <span style="font-style: italic;">American
Medical Informatics Association (AMIA-20) Summit on Clinical Research
Informatics</span>, Houston, USA. (<span style="font-weight: bold;">Best Student Paper Award</span>,<span style="font-size: 12pt; color: black;"> *Corresponding Autor</span>) PubMed <a target="_blank" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7233047/" style="outline-color: invert; outline-style: none; outline-width: 0px; color: rgb(100, 42, 143); font-family: arial,helvetica,clean,sans-serif; font-size: 13.4792px; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: right; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">32477657</a><br>
<span style="font-size: 12pt; color: black;"></span></li>
<li>Li, X. and Zhu, D. (2020). <a href="https://dongxiaozhu.github.io/ISBI_adversarial.pdf" target="_blank">Robust detection of adversarial attacks on medical images</a>. <span style="font-style: italic;">IEEE International Symposium on Biomedical Imaging (ISBI-20), </span>Iowa City, USA. <a href="https://github.com/xinli0928/COVID-Xray">Code</a> <a href="https://dblp.org/rec/conf/isbi/LiZ20.html?view=bibtex">DBLP</a>   </li><li>Li, X., Hect, J., Thompson, J. and Zhu, D. (2020). <a href="https://arxiv.org/abs/1906.03691" target="_blank">Interpreting age effects of human fetal brain from spontaneous fMRI using deep 3D convolutional neural networks</a>. <span style="font-style: italic;">IEEE International Symposium on Biomedical Imaging (ISBI-20)</span>, Iowa City, USA. <a href="https://github.com/xinli0928/COVID-Xray">Code</a> <a href="https://dblp.org/rec/conf/isbi/LiHTZ20.html?view=bibtex">DBLP</a> </li>
</ul><span style="font-family: Arial;">
2019</span><br style="font-family: Arial;">
<ul style="font-family: Arial;"><li><span style="font-size: 12pt; color: black;">Wang, Dong, M, Towner, E and Zhu, D</span> (2019) <a href="https://dongxiaozhu.github.io/BIBM19-ObesityRiskFactors.pdf" target="_blank">Prioritization of multi-level risk factors for obesity</a>. <span style="font-size: 12pt; color: black;">In the proceedings of 2019 IEEE <i style="">International Conference on Bioinformatics
and Biomedicine</i> (BIBM-19)</span>, 1065-1072. <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=Prioritization+of+Multi-Level+Risk+Factors+for+Obesity&amp;btnG=">GoogleScholar</a> <a href="https://dblp.org/rec/conf/bibm/WangDTZ19.html?view=bibtex">DBLP</a> <span style="font-size: 12pt; color: black;"></span></li>
  <li style="margin: 0px; padding: 0px; font-size: inherit;">Nezhad, MZ, Sadati, N, Yang, K and Zhu, D. (2019)<span> </span><a target="_blank" href="https://dongxiaozhu.github.io/ESA19-ActiveLearningSurvival.pdf" style="line-height: inherit; background-color: transparent; color: rgb(0, 0, 238); text-decoration: underline; cursor: pointer;">A deep active survival analysis approach for precision treatment recommendations</a><span style="color: rgb(0, 0, 238);">: application of prostate cancer.</span> <span style="font-style: italic;">Expert Systems with Applications</span>. Vol. 15, 16-26. <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=A+deep+active+survival+analysis+approach+for+precision+treatment+recommendations%3A+application+of+prostate+cancer&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:QaKAP0cguOsJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7do2ZI:AAGBfm0AAAAAXkdtwZJAmBxJXIWwg9uV8pWA3n5S4y-0&amp;scisig=AAGBfm0AAAAAXkdtwYUZrZyPsJ9-5GzpZsUPzZMoZ4W3&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a></li>
</ul><span style="font-family: Arial;">

2018</span><br style="font-family: Arial;"><ul style="font-family: Arial;">
  <li style="margin: 0px; padding: 0px; font-size: inherit;">Wang, L, Zhu, D, Towner, E and Dong, M (2018)<span>&nbsp;</span><a href="https://dongxiaozhu.github.io/BHI18-ObesityRiskFactors.pdf" style="line-height: inherit; background-color: transparent; color: rgb(0, 0, 238); text-decoration: underline; cursor: pointer;">Obesity risk factors ranking using multi-task learning</a>. <span style="font-style: italic;">IEEE Conference on Biomedical and Health Informatics</span> (IEEE-BHI 2018), Las Vegas, March, 2018. <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=Obesity+risk+factors+ranking+using+multi-task+learning&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:ACsBTE4-lU8J:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7fx-wY:AAGBfm0AAAAAXkf04wb2ilcC0p9SvEn4bN-pRfDj05lq&amp;scisig=AAGBfm0AAAAAXkf046ueDNYXxwlZF3sQRD-dukeTqO4U&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a></li>
  <li style="margin: 0px; padding: 0px; font-size: inherit;">Zheng, J, Gao, L, Zhang, H, Zhu, D, Wang, H, Gao, Q and Leung, V.<span>&nbsp;</span><a href="https://dongxiaozhu.github.io/AccessMaxMin.pdf" style="line-height: inherit; background-color: transparent; color: rgb(0, 0, 238); text-decoration: underline; cursor: pointer;">Joint energy management and interference coordination with Max-Min fairness in ultra-dense hetnets</a><span style="color: rgb(0, 0, 238);">.</span> <span style="font-style: italic;">IEEE Access</span>, Vol. 6, 32588-32600. <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=Joint+Energy+Management+and+Interference+Coordination+With+Max-Min+Fairness+in+Ultra-Dense+HetNets&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:cHKtwhsJhqoJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7fwRkM:AAGBfm0AAAAAXkf1XkPDUfLYxIOj0FEDfzVO4IKXIopp&amp;scisig=AAGBfm0AAAAAXkf1Xq8SgrpyvqpLZOQhbmohpK6HMaJI&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a></li>
  <li style="margin: 0px; padding: 0px; font-size: inherit;">Li, X and Zhu, D (2018)<span>&nbsp;</span><a href="https://dongxiaozhu.github.io/PRL18-FMR.pdf" style="line-height: inherit; background-color: transparent; color: rgb(0, 0, 238); text-decoration: underline; cursor: pointer;">Robust feature selection via l 2, 1 -norm in finite mixture of regression</a>. <span style="font-style: italic;">Pattern Recognition Letters</span>, 108 (2018) 15–22. <a href="https://scholar.google.com/scholar?q=Robust+feature+selection+via+l2%2C+1-norm+in+finite+mixture+of+regression.">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:PHcJN0o5J2IJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7fyCJQ:AAGBfm0AAAAAXkf3EJSBQcMd8w1aKglahs4ffbnmxhMd&amp;scisig=AAGBfm0AAAAAXkf3EEnN_xZOE4VtQE2RBF2N1TanNjxd&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a></li>
  <li><span style="font-size: 12pt; color: black;">Wang, L, Zhu, D and Dong, M</span> (2018) <a href="https://dongxiaozhu.github.io/SAM18-Clustering.pdf" target="_blank">Clustering over-dispersed data with mixed feature types</a>. <span style="font-style: italic;">Statistical Analysis and Data mining</span>, 2018;11:55–65. <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=Clustering+over-dispersed+data+with+mixed+feature+types&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:Dea4eW0YruoJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7fy1cw:AAGBfm0AAAAAXkf3zcwRYLOnBNyOKQFPs3Ldok9l9xDo&amp;scisig=AAGBfm0AAAAAXkf3zXcnUz5QVpLCA7aLwbNu7MbxtvUp&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a> <span style="font-size: 12pt; color: black;"></span></li>
<li><span style="font-size: 12pt; color: black;">Li, X, Zhu, D and Dong, M (2018) </span><a href="https://dongxiaozhu.github.io/PRL18-OverlappingLasso.pdf">Multinomial classification with class-conditional overlapping sparse feature groups</a>. <span style="font-style: italic;">Pattern Recognition Letters</span>, vol 101, Jan. 2018, pp 37-43 <a href="https://dongxiaozhu.github.io/CCSOGL%20Package.zip">Source Code</a> <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=Multinomial+classification+with+class-conditional+overlapping+sparse+feature+groups.&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:4hI6fSCpDaQJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7f9J3s:AAGBfm0AAAAAXkf4P3s8Yqykz_H8dcIJcjQuHPDDSF09&amp;scisig=AAGBfm0AAAAAXkf4P9qRNPp800ioxHiKf4JYK62wJ9lD&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a><span style="font-size: 12pt; color: black;"></span></li></ul><br style="font-family: Arial;"><span style="font-family: Arial;">2017

</span><ul style="font-family: Arial;"><li><span style="font-size: 12pt; color: black;">Wang, L, Acharya, L, Bai, C and Zhu, D (2017) <a href="https://dongxiaozhu.github.io/QB17-TranscriptomeAssembly.pdf">Transcriptome assembly strategies for precision medicine</a>. <span style="font-style: italic;">Quantitative Biology</span>, pp 1-11</span>, https://doi.org/10.1007/s40484-017-0109-2. <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=Transcriptome+assembly+strategies+for+precision+medicine&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:CFwZcKv5ABEJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7f9tPo:AAGBfm0AAAAAXkf4rPrO2It-apsyUNEtWVXyF4fnioty&amp;scisig=AAGBfm0AAAAAXkf4rJIzlD4KaM4omh0LL1vmJhyr6Mc0&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a><span style="font-size: 12pt; color: black;"></span></li><li><span style="font-size: 12pt; color: black;">Wang, L, Zhu, D*<span style="font-weight: bold;">, </span>Li, Y and Dong, M (2017) <a href="https://dongxiaozhu.github.io/ICMLA17.pdf">Modeling Over-dispersion for Network Data Clustering</a>.
In the proceeding of 16<sup>th</sup> IEEE <span style="font-style: italic;">International Conference on Machine
Learning and Application</span> (ICMLA’17). (<span style="font-weight: bold;">Best Paper Award Top 3 Finalist</span>, *Corresponding Autor) </span><a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=Modeling+Over-dispersion+for+Network+Data+Clustering&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:bueQ6sqXYEcJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7f8L0E:AAGBfm0AAAAAXkf5N0F95TrDTjtfo2A_zMs3f5bNVq0I&amp;scisig=AAGBfm0AAAAAXkf5N5QbajERkitEESjgoWYGOrL7eyvM&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a>
<span style="font-size: 12pt; color: black;"></span><span style="font-size: 12pt; color: black;"></span></li><li><span style="font-size: 12pt; color: black;">Nezhad, MZ, Zhu, D*, Yang, K
and Levy, P. (2017) </span><span style="font-size: 12pt; color: black;"><a target="_blank" href="https://dongxiaozhu.github.io/ICMLA17b.pdf">A Supervised Bi-Clustering Approach for
Precision Medicine</a>. In the proceeding of 16<sup>th</sup> IEEE <span style="font-style: italic;">International
Conference on Machine Learning and Application</span> (ICMLA’17). &nbsp;</span><span style="font-size: 12pt; color: black;">(<span style="font-weight: bold;">Best Poster Award Top 3 Finalist</span>, *Corresponding Autor) </span><a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=A+Supervised+Bi-Clustering+Approach+for+Precision+Medicine&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:h_a6BP8NkUcJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7f_WxU:AAGBfm0AAAAAXkf6QxXgzQwYUZEI7WfMCjInLx1n6uXD&amp;scisig=AAGBfm0AAAAAXkf6Q7RD8WUwjRLDjy4Ab4QSSlGVWZec&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a>
</li><li><span style="font-size: 12pt; color: black;">Li, X, Zhu, D and Levy, P (2017) <a href="https://dongxiaozhu.github.io/BIBM2017-AuxillaryRegularization.pdf">Predictive Deep Network with Leveraging
Clinical Measure as Auxiliary Task</a>. In the proceedings of 2017 IEEE <i style="">International Conference on Bioinformatics
and Biomedicine</i> (BIBM’17) </span><span style="font-size: 12pt; color: black;"></span><a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=Predictive+Deep+Network+with+Leveraging+Clinical+Measure+as+Auxiliary+Task&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:WHmdSZk2pmEJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7f_ov0:AAGBfm0AAAAAXkf6uv0rkQqWw73c_QQ1sGwNHMXJzBRf&amp;scisig=AAGBfm0AAAAAXkf6umrBRYTwbzHXD1KfeyqYea1ALS2z&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a><span style="font-size: 12pt; color: black;"> <o:p></o:p></span></li><li><span style="font-size: 12pt; color: black;">Wang, L, Li, Y, Zhou, J, Zhu, D and Ye, J (2017) <a href="https://dongxiaozhu.github.io/ICDM17-Survival.pdf">Multi-task
Survival Analysis</a>. In the proceedings of 2017 IEEE <i style="">International Conference on Data Mining</i> (ICDM’17) </span><a href="https://scholar.google.com/scholar?cluster=1662190289556401335&amp;hl=en&amp;as_sdt=0,23">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:t-wc_kJJERcJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7f-WNM:AAGBfm0AAAAAXkf7QNOQMiegqfP007_QOG2M20j5yx8u&amp;scisig=AAGBfm0AAAAAXkf7QDE1aaPCZHoHXPTeySdZtghqhbj7&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en&amp;scfhb=1">BibTeX</a><span style="font-size: 12pt; color: black;"> </span><span style="font-size: 12pt; color: black;"><o:p></o:p></span>
</li><li>Li, X, Zhu, D, Dong, M, Nezhad, MZ and Levy, P (2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28815129" target="_blank">SDT:
A Tree Method for Detecting Patient Subgroups with Personalized Risk
Factors</a>. In the proceedings of 2017 American Medical Informatics
Association (AMIA) <span style="font-style: italic;">Summit on Clinical Research Informatics</span>, San Francisco, March 2017. <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=SDT%3A+A+Tree+Method+for+Detecting+Patient+Subgroups+with+Personalized+Risk+Factors&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:PZD4nIQ51a8J:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7f-kVI:AAGBfm0AAAAAXkf7iVL-KkAOmu-yKgtwoBzppVC4HZ_1&amp;scisig=AAGBfm0AAAAAXkf7iRMK8v4QB50tvvRKZ86hPdrrBNM8&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a>
</li>
</ul><span style="font-family: Arial;">
2016</span><br style="font-family: Arial;">
<ul style="font-family: Arial;"><li>Nezhad, MZ, Zhu, D, Li, X, Yang, C and Levy, P (2016)&nbsp; <a href="https://dongxiaozhu.github.io/BIBM16-SAFS.pdf">SAFS: A
Deep Feature Selection Approach for Precision Medicine</a>. In the
proceedings
of 2016 IEEE Inernational Conference on Bioinformatics and Biomedicine
(IEEE BIBM 2016). <a href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C23&amp;q=SAFS%3A+A+Deep+Feature+Selection+Approach+for+Precision+Medicine&amp;btnG=">GoogleScholar</a> <a href="https://scholar.googleusercontent.com/scholar.bib?q=info:RGb6qQzdsywJ:scholar.google.com/&amp;output=citation&amp;scisdr=CgWZAe5wEN2Bt7f5HtU:AAGBfm0AAAAAXkf8BtU91O0BTXm28QoPxj9ycZ3o426N&amp;scisig=AAGBfm0AAAAAXkf8BkFtIQdKI8yrPiGGJurK9B9qM_nI&amp;scisf=4&amp;ct=citation&amp;cd=-1&amp;hl=en">BibTeX</a>


  </li>

  <li>Xu, H, Dong, M, Zhu, D, et al. (2016) <a href="https://www.researchgate.net/publication/305773577_Text_Classification_with_Topic-based_Word_Embedding_and_Convolutional_Neural_Networks" target="_blank">Text Classification with
Topic-based Word Embedding and Convolutional Neural Networks</a>. In the proceedings of 2016 ACM Conference on
Bioinformatics, Computational Biology and Health Informatics (ACM BCB 2016).</li>

  <li>Wang, L, Zhu, D, Li, Y and Dong, M. (2016) <a href="http://link.springer.com/chapter/10.1007%2F978-3-319-38782-6_2" target="_blank">Poisson-Markov Mixture
Model and Parallel Algorithm for Binning Massive and Heterogeneous DNA
Sequencing Reads</a>. In the Series of <span style="font-style: italic;">Lecture Notes in Computer
Science</span> (ISBRA 2016).</li>

  <li>Almomani, R, Dong, M and Zhu, D. (2016) <a href="http://ieeexplore.ieee.org/document/7552909/" target="_blank">Bayesian Hierarchical
Appearance Model for Robust Object Tracking</a>. In the Proceeding of
International Conference on Multimedia and Expo (ICME 2016). <br>
</li>
  <li>Almomani, R, Dong, M and Zhu, D. (2016) <a href="http://link.springer.com/article/10.1007/s00521-016-2280-1" target="_blank">Object Tracking via Dirichlet
Process-based Appearance Models</a>. <span style="font-style: italic;">Neural Computing and Applications</span>, in press. DOI: 10.1007/s00521-016-2280-1.</li><li>Hou, J., Acharya, L., Zhu, D. and Chen, J. (2016) <a href="https://academic.oup.com/bfg/article-abstract/15/2/95/2450501?redirectedFrom=PDF" target="_blank">An overview of bioinformatics methods for modeling biological pathways in yeast</a>. <span style="font-style: italic;">Briefings in Functional Genomics</span>, 15(2), 95-108.&nbsp; </li>


</ul><h4 style="font-family: Arial;">Selected Federal Funding:</h4><ol style="font-family: Arial;">
  <li><span style="font-weight: bold;">NIH/R61HD105610</span>:
Severity Predictors Integrating salivary Transcriptomics and proteomics
with Multi neural network Intelligence in SARS-CoV2 infection in
Children (SPITS MISC) 01/01/2021-12/31/2022. (<span style=""><span style="font-weight: bold;">Principal Investigator</span></span>)<br>
    <span style="font-weight: bold;"></span></li>
  <li><span style="font-weight: bold;">NSF/CNS 2043611:</span>
SCC-CIVIC-PG Track A: Leveraging AI-assist Microtransit to Ameliorate
Spatiotemporal Mismatch between Housing and Employment.
01/01/2021-06/30/2021. (<span style=""><span style="font-weight: bold;">Principal Investigator</span></span>) <br>
<span style="font-weight: bold;"></span></li>
  <li><span style="font-weight: bold;">NSF/CCF 1637312:</span> S&amp;CC:
Promoting a Healthier Urban Community: Prioritization of Risk Factors
for the Prevention and Treatment of Pediatric Obesity. 09/01/2016-08/31/2018. <span></span><span style="">(<span style="font-weight: bold;">co-Principal Investigator</span>)</span> <br>
<span style="font-weight: bold;"></span></li>

  <li><span style="font-weight: bold;">NSF/IIS 1724227:</span> <span itemprop="name">S&amp;AS:
INT: Autonomous Battery Operating System (ABOS): An Adaptive and
Comprehensive Approach to Efficient, Safe, and Secure Battery System
Management</span>. 09/01/2017-08/31/2021. (<span style="font-weight: bold;">Senior Personnel</span>) <span style="font-weight: bold;"></span></li><li><span style="font-weight: bold;">NSF/CCF</span> <span style="font-weight: bold;">1451316</span>:&nbsp; EAGER: A novel algorithmic framework for discovering
subnetworks from big biological data. 08/15/2014-08/14/2017. (<span style="font-weight: bold;">Principal
Investigator</span>)<br>

    <span><span style="font-weight: bold;"></span></span></li>
  <li><span style="font-weight: bold;">NIH/R21LM010137</span>:&nbsp; R21.A new informatics paradigm for reconstructing signaling
pathways in human disease. 09/2009 – 08/2012. (<span style="font-weight: bold;">Principal Investigator</span>)<br>

<span><span style="font-weight: bold;"></span></span></li><li><span><span style="font-weight: bold;">NSF/CCF</span> <span style="font-weight: bold;">0939108</span>: CPATH: A verification based learning model that enriches CS and related undergraduate programs. 10/01/2009-09/30/2012 </span><span style="">(<span style="font-weight: bold;">co-Principal Investigator</span>)</span></li>
</ol><h4 style="font-family: Arial;"><span style="font-weight: bold;">Current Students Whose Research under My Supervision</span>:<br>
</h4>
<ul style="font-family: Arial;"><li>Xiangrui Li (CS Ph.D. candidate)</li><li>Deng Pan (CS Ph.D.&nbsp;candidate)</li><li>Xin Li (CS Ph.D.&nbsp;candidate)</li>
  <li>Qiang Yao (CS Ph.D. pre-candidate)</li>
<li>Chengyin Li (CS Ph.D. pre-candidate)</li>
  <li>Supriya T.S. Kumar (CS Ph.D. pre-candidate)</li><li><span style="color: rgb(0, 0, 0); font-family: Arial; font-size: medium; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; display: inline ! important; float: none;">Brian Bruns (CS Master thesis student)</span></li>
</ul><span style="font-family: Arial;"><span style="font-weight: bold;">Past Students</span>:<br></span>
<ul style="font-family: Arial;">
  
  <li>Najibesadat Sadatijafarkalaei (M.S. in 2020). <span style="color: rgb(0, 0, 0); font-family: Arial; font-size: medium; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; display: inline ! important; float: none;">First position: Clinical Data Scientist @ <a href="https://www.medschool.umaryland.edu/" target="_blank">Univ. of Maryland Medical School</a><br>
    </span></li>
  <li><span style="color: rgb(0, 0, 0); font-family: Arial; font-size: medium; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; display: inline ! important; float: none;">Xue Zhang (M.S. in 2020). First position: Cloud Developer @<span>&nbsp;</span></span><a href="https://www.quickenloans.com/" target="_blank" style="font-family: Arial; font-size: medium; font-style: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px;">Quicken Loans Inc.</a></li>

  <li>Milad Zafar Nezhad (Ph.D. 2018). First position: Data Scientist @ <a href="https://www.ford.com/" target="_blank">Ford Motor Company</a><br>
  </li>

  <li>Mohammad Hessam Olya (M.S. 2018) First position: Data Scientist @ <a href="https://www.ford.com/" target="_blank">Ford Motor Company</a></li>
  <li>Narjes S. Movahedi (Ph.D. 2015). First Position: Software Engineer @ <a href="http://www.mathworks.com/" target="_blank">MathWorks</a><br>
  </li>
<li>Nan Deng (Ph.D. 2014). First Position: Senior Research Bioinformatician @ <a target="_blank" href="http://www.cedars-sinai.edu/Patients/Programs-and-Services/Samuel-Oschin-Comprehensive-Cancer-Institute-/Research-and-Clinical-Trials/Biostatistics-and-Bioinformatics-Research-Center/Current-Members.aspx">Cedars Sinai Medical Center</a></li><li>Thair Judeh (Ph.D. 2014).&nbsp;First Position:: Lecturer @ <a target="_blank" href="http://engineering.wayne.edu/cs/">Wayne State University Computer Science Department</a></li><li>Lipi R. Acharya (Ph.D. 2011). First Position: Senior Computational Biologist @ <a target="_blank" href="http://www.dowagro.com/">Dow AroSciences</a>&nbsp;</li><li>Jingwei Zhang (M.S. 2017). Fisrt Position: Software Engineer @ New Jersey&nbsp;</li><li>Kristen Johnson (M.S. 2012) First Position: PhD Student @ <a target="_blank" href="https://www.cs.purdue.edu/">Purdue University Computer Science Department</a></li><li>Guorong Xu (M.S. 2011) First Position: Computational Biologist @ <a target="_blank" href="http://www.baylorhealth.com/Pages/Default.aspx">Baylor Medical Center Dallas</a>&nbsp;</li><li>Patrick Trahan (M.S. 2009)&nbsp;First Position: IT Professional @ <a target="_blank" href="http://www.lsuhsc.edu/">Lousiana State Univeristy Helath Science Center</a></li><li>Sichu Li (M.S. 2009) First Position: Staff Scientist @ <a target="_blank" href="http://www.mitre.org/">MITRE Corperation</a>&nbsp;</li>
</ul>
<span style="font-weight: bold; font-family: Arial;"></span>
<h4 style="font-family: Arial;"><span style="font-weight: bold;">Software</span>:<br>

    
    </h4>
<ol style="font-family: Arial;">
<li><a target="_blank" href="http://bioconductor.org/packages/devel/bioc/html/CORREP.html">CORREP</a>:
          a Bioconductor package to estimate correlation between two
          variables with replicates</li><li><a href="http://cran.r-project.org/src/contrib/Archive/GeneNT/" target="_blank">GeneNT</a>: a R package to estimate
          co-expression gene networks</li><li><a href="http://sammate.sourceforge.net/" target="_blank">SAMMate</a>:
          a GUI system for transcriptoime assembly and quantification
          using RNA-Seq (<span style="font-weight: bold;">A popular sotware</span> in its field:<span style="font-weight: bold;"> 7000+ downloads, </span>used by researchers from <span style="font-weight: bold;">all over the world</span> and <span style="font-weight: bold;">cited by</span> papers published in <span style="font-style: italic;">Nature Genetics</span>, <span style="font-style: italic;">PNAS</span>, <span style="font-style: italic;">Genome Research</span> etc.)<br>

</li><li><a href="http://code.google.com/p/teak/" target="_blank">TEAK</a>:
          a GUI system for Topological Enrichment Analysis frameworK</li><li><a href="http://dsplicetype.sourceforge.net/" target="_blank">dSpliceType</a>: a Java-based tool to detect various types of differential splicing events using RNA-Seq&nbsp;</li>
</ol>
<span style="font-family: Arial;"><span style="font-weight: bold;"></span></span><h4 style="font-family: Arial;">Synergistic
        Activities:</h4>
    <ul style="font-family: Arial;">
      <li>National Science Foudation Panelist, 2009(2),&nbsp;2011, 2013, 2014 (mail reviewer), 2015, 2017, 2020. &nbsp;</li><li>National Institute of Health ACE Study Section, 2009-2011, multiple special emphasis panels 2011-2020.</li>
      
  <li>Associate Editor: <a href="http://www.biomedcentral.com/bmcgenomics"><span style="font-style: italic;">BMC Genomics</span> </a><br>

  </li>
  <li>Associate Editor:
          <a href="http://www.frontiersin.org/genetics"><span style="font-style: italic;">Frontiers in Genetic</span>s</a> <br>

    </li>
  <li>Academic Editor: <a href="http://www.plosone.org/"><span style="font-style: italic;">PLoS ONE</span></a><span style="font-style: italic;">&nbsp;</span></li><li>Associate Editor (Bioinformatics):<span style="font-style: italic;"> </span><a href="http://www.springer.com/life+sciences/journal/11427"><span style="font-style: italic;">Science China - Life Sciences</span></a></li>
  <li>Editorial Board Member: <a target="_blank" href="https://www.nature.com/srep/"><span style="font-style: italic;">Scientif Reports</span></a><span style="font-style: italic;"></span></li><li>Editorial Board Member:&nbsp;<a href="http://www.inderscience.com/jhome.php?jcode=IJCBDD"><span style="font-style: italic;">International Journal of Computational Biology and Drug Design</span></a></li>
  <li><a href="http://www.inderscience.com/jhome.php?jcode=IJCBDD"><span style="font-style: italic;"></span></a>Guest Editor: <span style="font-style: italic;">IEEE/ACM transaction on Computational Biology and Bioinformatics (TCBB)</span>; <span style="font-style: italic;">BMC Bioinformatics</span><span style="font-style: italic;"></span>. <br>

    </li>

      <li>Conference
technical committee: ICML-21, IJCAI-21, NuerIPS-20, AAAI-20, IJCAI-20, ACL-20, AMIA-20,
MICCAI-20, BIBM 2011-19, BCB 2013-18, InCoB'12-19 </li>
      <li>Program/Session/Workshop
chair: ACM-BCB'15, ICIBM'14, IEEE-ICMLA'14, ACM-BIOKDD'14, ICIBM'13,
ACM-BCB'14, ACM-BCB'13, ACM-BCB'12, IEEE-BIBM'13, IEEE-BIBM'12,
ICIBM'12, IEEE-BIBM'11,&nbsp;ENAR’07.</li>
      <li>Journal reviewer: <span style="font-style: italic;">Bioinformatics, Briefings in Bioinformatics, Nucleic Acids Research, Genome
            Biology, Genome Medicine, BMC Genomics, BMC Bioinformatics, BMC Systems
            Biology, Biometrics, </span><span style="font-style: italic;">Statistics in Medicine, </span><span style="font-style: italic;">PLoS Genetics, </span><span style="font-style: italic;">PLoS
ONE, Biology Direct, IEEE transaction on Fuzzy Systems (TFS),
Computational Biology and Bioinformatics (TCBB),&nbsp; Knowledge and
Data Engineering (TKDE), Sustainable Computing (TSUSC), Computers (TC),
Medical Imaging (TMI), Artifical Intelligence (TAI), IEEE Signal
Processing Letters, </span><span style="font-style: italic;">Physics Review.<br>
</span></li>
    </ul>
    <h4 style="font-family: Arial;">Classroom Teaching (Computer Science Department at Wayne State University):</h4>
    <ul style="font-family: Arial;"><li>CSC 8800 Seminar in Machine Learning and Artificial Intelligence. Winter 2021<br>
</li>
  <li>CSC 8860 Seminar Topics in Computer Vision and Pattern Recognition. Fall 2017</li>
<li>CSC 7825 Machine Learning. Fall 2014, Fall 2015, Fall 2016, Winter 2019, Winter 2020, Winter 2021<br>
</li>
  <li>CSC 6580 Design and Analysis of Algorithms. Winter 2015, Winter 2016, Winter 2017 <br>
 </li>
  <li>CSC 5825 Intro. to Machine Learning and Applications. Winter 2017, Fall 2017, Fall 2018, Fall 2019, Fall 2020<br>
  </li>

<li>CSC 5991 Special Topics in Comp. Sci. Fall 2012, Fall 2013, Winter 2014 , Winter 2015, Winter 2016 </li>
      <li>CSC 2110 Comp.
          Sci. I (C++ Programming).
          Fall 2011, Winter 2012, Winter 2013, Fall 2013&nbsp;</li></ul><span style="font-family: Arial; font-weight: bold;"></span><!-- Start of StatCounter Code -->
    <script type="text/javascript">
var sc_project=7244385; var sc_invisible=1; var sc_security="0dbaa548"; </script>
    <script type="text/javascript" src="./index_files/counter.js.download"></script>
    <noscript><div class="statcounter"><a title="tumblr
statistics" href="http://statcounter.com/tumblr/" target="_blank"><img class="statcounter" src="http://c.statcounter.com/7244385/0/0dbaa548/1/" alt="tumblr
statistics"></a></div></noscript>
    <!-- End of StatCounter Code -->
  </body></html>